pkg/subctl/cmd/validate_cni.go

```diff
@@ -78,4 +94,123 @@ func validateCNIInCluster(clusterName string, submariner *v1alpha1.Submariner) {
 		submariner.Status.NetworkPlugin)
 	status.QueueSuccessMessage(message)
 	status.End(cli.Success)
+
+	validateCalicoIPPoolsIfCalicoCNI(config)
+}
+
+func doesCalicoConfigMapExist(clientSet kubernetes.Interface) (*v1.ConfigMap, error) {
+	cmList, err := clientSet.CoreV1().ConfigMaps(metav1.NamespaceAll).List(metav1.ListOptions{})
```

Comment:
Since we're looking for a CM with a specific name, would `Get` with `NamespaceAll` also work?

```suggestion
	cm, err := clientSet.CoreV1().ConfigMaps(metav1.NamespaceAll).Get("calico-config", ...)
```

Reply:
I tried that and it did not work. It looks like we cannot call Get() with NamespaceAll.